CalibTracker/SiPixelConnectivity/test/SiPixelFedCablingMapWriter.cc

```diff
@@ -70,7 +70,7 @@ void SiPixelFedCablingMapWriter::analyze(const edm::Event& iEvent, const edm::Ev
 void SiPixelFedCablingMapWriter::beginJob() {}
 
 void SiPixelFedCablingMapWriter::endJob() {
-  SiPixelFedCablingMap* result = new SiPixelFedCablingMap(cabling);
+  const auto& result = SiPixelFedCablingMap(cabling);
```

Comment:
Maybe i'm simply not used to that syntax, which i'm sure compiles... but is it potentially leading to an invalid read? I mean, here we construct an object of type SiPixelFedCablingMap with the constructor taking one param - does it stay on the stack or it dies ? In the second case, the reference to it (the variable result) could point to something no longer in memory? Maybe it is just my bias, but having doubt I would rather go for:
SiPixelFedCablingMap result(cabling);

Reply:
@ggovi done in the last push.